incident-reports/_posts/2022-08-01-august.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title: September 2022 (Placeholder)
```

Comment:
> The incident reports shouldn't be date based (e.g. August 2022) as we don't expect to produce them regularly.

This is just a matter of choosing a more specific title (and ideally making the file name more descriptive). Otherwise, there’s no systemic issue here.